tests/tests/const_global.c

```diff
@@ -8,7 +8,7 @@
 //     jit-state: stop-tracing
 //     --- Begin jit-pre-opt ---
 //     ...
-//     define internal void @__yk_compiled_trace_0(%YkCtrlPointVars* %0, i64* %1, i64 %2) {
+//     define internal i8 @__yk_compiled_trace_0(%YkCtrlPointVars* %0, i64* %1, i64 %2) {
```

Comment:
What's the `i8` return again?

Reply:
One of these: https://github.com/ykjit/yk/pull/500/files#diff-9fa85f3ab0ab08c2bb4337dd17d47605f520c2780158e50e66eff7cd9dc1b29bR42

```rust
const TRACE_RETURN_SUCCESS: u8 = 0;
const TRACE_GUARDFAIL_CONTINUE: u8 = 1;
const TRACE_GUARDFAIL_RETURN: u8 = 2;
```

Though this is likely subject to change at some later stage.